### PR TITLE
sdk-dev: Improve SDK Interface docs

### PR DESCRIPTION
**Pull request by rhcarvalho**
- Update first sentence to match other interfaces
- Add missing 'Attributes' section header
- Clarify version format
- Clarify package format
- Update example to contain valid JSON
- Fix typo in example SDK name 'javscript' => 'javascript'

### CODE REVIEW

File: src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md [hunk 1]
```diff
@@ -25,28 +28,36 @@ capture and transmit the event.
 
 : _Optional_. A list of packages that were installed as part of this SDK or the
   activated integrations. Each package consists of a `name` in the format
-  `source:identifier` and a semver `version`. If the source is a git repository,
-  a checkout link and git reference (branch, tag or SHA) should be used.
+  `source:identifier` and `version`.  
+  If the source is a Git repository, the source should be `git`, the
+  `identifier` should be a checkout link and the `version` should be a Git
+  reference (branch, tag or SHA).
```
Comment (rhcarvalho) [inline src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md:34]: I tried to update this to the best of my understanding. Please check if it is accurate.

File: src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md [hunk 2]
```diff
@@ -25,28 +28,36 @@ capture and transmit the event.
 
 : _Optional_. A list of packages that were installed as part of this SDK or the
   activated integrations. Each package consists of a `name` in the format
-  `source:identifier` and a semver `version`. If the source is a git repository,
-  a checkout link and git reference (branch, tag or SHA) should be used.
+  `source:identifier` and `version`.
```
Comment (rhcarvalho) [inline src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md:31]: I noticed the Go SDK doesn't follow this format.

What are the valid values for `source`? Is it free-form? Is it used for anything?

File: src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md [hunk 3]
```diff
@@ -3,18 +3,21 @@ title: SDK Interface
...
+: **Required**. The name of the SDK. The format is `sentry.ecosystem[.flavor]`
   where the _flavor_ is optional and should only be set if it has its own SDK.
 
 `version`:
 
-: **Required**. The semantic version of the SDK. The version should always be
-  sent without a `v` prefix.
+: **Required**. The version of the SDK. It should have the [Semantic
+  Versioning](https://semver.org) format `MAJOR.MINOR.PATCH`, without any
+  prefix.
```
Comment (jan-auer) [inline src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md:20]: Maybe keep `v` as an example here, just to be explicit.

File: src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md (hunk 2 above)
Comment (jan-auer) [reply src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md:31]: From ingestion perspective, this is completely freeform, but I'm guessing that we might need some recommendation here.

File: src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md (hunk 3 above)
Comment (rhcarvalho) [reply src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md]: Done.

**Comment by rhcarvalho** [comment]
@jan-auer did you have some particular concern that @HazAT can clarify?

Otherwise the PR as is doesn't change the content in any way as to change interpretation. The open questions are just further improvements we don't have to block on:

- Clarify valid values for `source`.
- Clarify how to specify dependencies coming from `git`.

Those are valid questions, and they were already in the minds of readers to begin with, not something we're introducing or changing here.

**Comment by jan-auer** [comment]
> did you have some particular concern that @HazAT can clarify?

You had a few open questions, which he's the better person to answer. Ultimately, I'd prefer @HazAT to sign off on this, since this is his domain. No objections from my end.

File: src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md [hunk 4]
```diff
@@ -3,18 +3,21 @@ title: SDK Interface
...
-capture and transmit the event.
+The SDK Interface describes the Sentry SDK and its configuration used to capture
+and transmit an event.
+
+## Attributes
 
 `name`:
 
-: **Required**. The name of the SDK. Its format is `sentry.ecosystem[.flavor]`
+: **Required**. The name of the SDK. The format is `sentry.ecosystem[.flavor]`
```
[the remark this answers is not in the document]
Comment (HazAT) [reply src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md:13]: valid point, not against that.

**Pull request merged**